### PR TITLE
Feature/make explore page responsive

### PR DESCRIPTION
**Pull request by Knochenmark**

### CODE REVIEW

File: src/app/app-routing.module.ts [hunk 1]
```diff
@@ -11,7 +11,7 @@ import { UserComponent } from './shared/user/user.component';
 
 const routes: Routes = [
   { path: '', redirectTo: '/explore', pathMatch: 'full' },
-  { path: 'settings', loadChildren: () => import('./settings/settings.module').then(m => m.SettingsModule) },
+  { path: 'settings/:id', loadChildren: () => import('./settings/settings.module').then(m => m.SettingsModule) },
```
Comment (Knochenmark) [inline src/app/app-routing.module.ts]: why we need a settingsId?

File: src/app/mobile-toolbar/mobile-toolbar.component.styl [hunk 2]
```diff
@@ -1,9 +1,14 @@
...
   margin-left 0.5rem
 
-.spacer
-  flex 1 1 auto
-
 .toolbar
-  border-bottom 2px solid #d8f3dc
-  color rgba(0, 0, 0, 0.87)
\ No newline at end of file
+  color rgba(0, 0, 0, 0.87)
```
Comment (Knochenmark) [inline src/app/mobile-toolbar/mobile-toolbar.component.styl]: we should use a variable instead

File: src/app/app-routing.module.ts (hunk 1 above)
Comment (mbos2) [reply src/app/app-routing.module.ts]: settings/:userID

File: src/app/mobile-toolbar/mobile-toolbar.component.styl [hunk 3]
```diff
@@ -1,9 +1,14 @@
...
+
+.menu-icons
+  display flex
+  gap 5px
```
Comment (Knochenmark) [inline src/app/mobile-toolbar/mobile-toolbar.component.styl]: not sure flex gap is supported everywhere already

**Review by Knochenmark — COMMENTED** [review]
see comments

File: src/app/app-routing.module.ts (hunk 1 above)
Comment (Knochenmark) [reply src/app/app-routing.module.ts]: I think what we want is base/:userId/settings

**Pull request merged**